disco_aws_automation/disco_rds.py

```diff
@@ -137,29 +284,93 @@ def get_instance_parameters(self, env_name, database_name):
+        try:
+            return self.client.describe_db_instances(DBInstanceIdentifier=instance_identifier)
+        except botocore.exceptions.ClientError:
+            return None
+
+    def create_db_parameter_group(self, db_parameter_group_name, db_parameter_group_family):
         """
-        Returns the intranet security group id for the VPC for the current environment
+        Creates a new DB parameter group. Used to set customized parameters.
+
+        A DB parameter group is initially created with the default parameters for the
+        database engine used by the DB instance.
+        To provide custom values for any of the parameters, you must modify the group after creating it.
         """
-        security_groups = self.disco_vpc_sg_rules.get_all_security_groups_for_vpc()
-        for security_group in security_groups:
-            if security_group.get('Tags'):
-                tags = tag2dict(security_group['Tags'])
-                if tags.get("meta_network") == "intranet":
-                    return security_group['GroupId']
+        self.client.create_db_parameter_group(
+            DBParameterGroupName=db_parameter_group_name,
+            DBParameterGroupFamily=db_parameter_group_family,
+            Description='Custom params-{0}'.format(db_parameter_group_name))
 
-        raise RuntimeError('Security group for intranet meta network is missing.')
+    def modify_db_parameter_group(self, db_parameter_group_name, parameters):
+        """
+        Modifies the parameters of a DB parameter group.
 
-    def update_cluster_by_id(self, database_identifier):
-        """Update a RDS cluster by its database identifier"""
-        self.update_cluster(self._get_database_name(self.vpc_name, database_identifier))
+        Submit a list of the following:
+        ('ParameterName', 'ParameterValue', 'Description', 'Source',
+         'ApplyType', 'DataType', 'AllowedValues', 'IsModifiable',
+         'MinimumEngineVersion', 'ApplyMethod')
+        """
+        for parameter in parameters:
+            keep_trying(RDS_STATE_POLL_INTERVAL,
+                        self.client.modify_db_parameter_group,
+                        DBParameterGroupName=db_parameter_group_name,
+                        Parameters=[{'ParameterName': parameter[0],
+                                     'ParameterValue': parameter[1],
+                                     'Description': 'Description',
+                                     'Source': 'engine-default',
+                                     'ApplyType': 'static',
+                                     'DataType': 'string',
+                                     'AllowedValues': 'somevalues',
+                                     'IsModifiable': True,
+                                     'MinimumEngineVersion': 'someversion',
+                                     'ApplyMethod': 'pending-reboot'}])
+
+    def recreate_db_parameter_group(self, env_name, database_name, db_parameter_group_name,
```

Comment:
I do not understand why we have `create_db_parameter_group`, `modify_db_parameter_group`, and `recreate_db_parameter_group`. Can't we implement *create* and *recreate* in the same function?